tasks/20230731-stakeless-gauge-checkpointer/readme.md

```diff
@@ -0,0 +1,7 @@
+# 2023-05-31 - L2 Gauge Checkpointer
```

Comment:
```suggestion
# 2023-07-31 - Stakeless Gauge Checkpointer
```
Also describe the reason for the name change? (e.g., clarify that these are on L2)